markdown/explore-dmr/explore-dmr.md

```diff
@@ -0,0 +1,295 @@
+author: Jamieson Walker
+summary:
+id: explore-dmr
+tags:
+categories: solace,dmr
+environments: Web
+status: Published
+feedback link: https://github.com/SolaceDev/solace-dev-codelabs/blob/master/markdown/explore-dmr/explore-dmr.md
+
+
+# DMR (Event Mesh) Exploration Procedures
+
+## Objectives
+Duration: 0:05:00
+
+### Prerequisites
+1. 2+ Solace PubSub+ Brokers deployed in environments where they are visable to eachother over a network. 
+2. Access to the Solace Broker WebUI for all Brokers via administrator credentials. 
+
+<br>**⚠️ A note on security: The brokers must be secured with a TLS certificate. If you are using a Solace PubSub+ Docker image please check out [my blog on quickly securing the broker with a self-signed TLS certificate.](SomeblogIhaveNotWrittenYet)**
```

Comment:
Once blog is up we will update codelab here

Reply:
I think I will pull this out as a separate PR. I have not had a chance to write up self signed cert creation for the broker.